content/systems/filecoin_token/payment_channels/_index.md

```diff
@@ -2,40 +2,112 @@
 title: Payment Channels
 weight: 2
 bookCollapseSection: true
-dashboardAudit: 0
-dashboardState: wip
+dashboardAudit: 1
+dashboardState: Stable
 dashboardInterface: wip
 ---
 
 # Payment Channels
 ---
 
-Payment Channels are used in the Filecoin [Retrieval Market](retrieval_market) to enable efficient off-chain payments and accounting between parties for what is expected to be series of microtransactions, specifically those occurring as part of retrieval market data retrieval.
+Payment Channels are used in the Filecoin [Retrieval Market](retrieval_market) to enable efficient off-chain payments and accounting between parties for what is expected to be a series of microtransactions, as these occur during data retrieval.
 
-Note that the following provides a high-level overview of payment channels and an accompanying interface. The lotus implementation of [vouchers](https://github.com/filecoin-project/lotus/blob/master/chain/types/voucher.go) and [payment channels](https://github.com/filecoin-project/lotus/tree/master/paych) are also good references.
+In particular, given that there is no proving method provided for the act of sending data from a provider (miner) to a client, there is no trust anchor between the two. Therefore, in order to avoid mis-behaviour, Filecoin is making use of payment channels in order to realise a step-wise "data transfer <-> payment" relationship between the provider and the recipient of data. Clients issue requests for data that miners are responding to. The miner is entitled to ask for interim payments, the volume-oriented interval for which is agreed in the Deal phase. In order to facilitate this process, the Filecoin client is creating a payment channel once the provider has agreed on the proposed deal. The client should also lock in the payment channel monetary value equal to the one needed for retrieval of the entire block of data requested. Every time a provider is completing transfer of the pre-specified amount of data, they can request a payment. The client is responding to this payment with a voucher which the provider can redeem (immediately or later).
+
+
+Redeeming a voucher is not transferring funds from the payment channel to the provider's account, that is, it is not an on-chain operation. Instead, redeeming a voucher denotes the fact that data-volume worth of `voucher_val` has been transferred to the client and the corresponding value is locked on the provider's side of payment channel. It is not until the voucher is submitted on-chain through `Collect` that the funds are dispatched to the provider's account.
```

Comment:
"Redeeming a voucher is not transferring funds from the payment channel to the provider's account"

True -- no transfer happens until Collect is called.

"that is, it is not an on-chain operation."

False, or at least confusing in my mind.

There are three on-chain operations:
UpdateChannelState = how you submit vouchers
Settle - to begin closing a channel
Collect- When funds are actually transfered from client to provider

Redeem in my mind means submitting vouchers to the chain through UpdateChannelState

I think this entire paragraph is ultimately confusing.

Reply:
the process is:
Client creates a voucher -> sends to Provider
Provider -> saves the voucher locally
Either immediately or later, the Provider "redeems" the voucher by submitting it to the chain, calling UpdateChannelState (in the current software, it's always later -- our retrieval code does not call UpdateChannelState -- it has to be done manually from the CLI)
Client or Provider -> Calls settle
12 hour period begins
If the provider has not already done so, they should now submit the vouchers to the chain (Lotus now does this automatically)
12 hour period ends
Client or Provider -> calls Collect
Funds are transferred to Provider, any unclaimed balance goes back to Client